packages/in_app_purchase_tizen/lib/src/billing_manager_wrappers/billing_manager_wrapper.g.dart

```diff
@@ -0,0 +1,172 @@
+    };
+
+ProductsListApiResult _$ProductsListApiResultFromJson(
+        Map<String, dynamic> json) =>
+    ProductsListApiResult(
+      cPStatus: json['CPStatus'] as String? ?? '',
+      cPResult: json['CPResult'] as String? ?? '',
+      checkValue: json['CheckValue'] as String? ?? '',
+      totalCount: json['TotalCount'] as int? ?? 0,
+      itemDetails: (json['ItemDetails'] as List<dynamic>?)
+              ?.map((e) => ItemDetails.fromJson(e as Map<String, dynamic>))
+              .toList() ??
+          [],
+    );
+
+Map<String, dynamic> _$ProductsListApiResultToJson(
+        ProductsListApiResult instance) =>
+    <String, dynamic>{
+      'CPStatus': instance.cPStatus,
+      'CPResult': instance.cPResult,
+      'TotalCount': instance.totalCount,
+      'CheckValue': instance.checkValue,
+      'ItemDetails': instance.itemDetails,
+    };
+
+ItemDetails _$ItemDetailsFromJson(Map<String, dynamic> json) => ItemDetails(
+      seq: json['Seq'] as int? ?? 0,
+      itemID: json['ItemID'] as String? ?? '',
+      itemTitle: json['ItemTitle'] as String? ?? '',
+      itemDesc: json['ItemDesc'] as String? ?? '',
+      itemType: json['ItemType'] as int? ?? 0,
+      price: json['Price'] as num? ?? 0,
+      currencyID: json['CurrencyID'] as String? ?? '',
+    );
+
+Map<String, dynamic> _$ItemDetailsToJson(ItemDetails instance) =>
+    <String, dynamic>{
+      'Seq': instance.seq,
+      'ItemID': instance.itemID,
+      'ItemTitle': instance.itemTitle,
+      'ItemDesc': instance.itemDesc,
+      'ItemType': instance.itemType,
+      'Price': instance.price,
+      'CurrencyID': instance.currencyID,
+    };
+
+ProductSubscriptionInfo _$ProductSubscriptionInfoFromJson(
+        Map<String, dynamic> json) =>
+    ProductSubscriptionInfo(
+      paymentCycle: json['PaymentCycle'] as int? ?? 0,
+      paymentCycleFrq: json['PaymentCycleFrq'] as int? ?? 0,
+      paymentCyclePeriod: json['PaymentCyclePeriod'] as String? ?? '',
+    );
+
+Map<String, dynamic> _$ProductSubscriptionInfoToJson(
+        ProductSubscriptionInfo instance) =>
+    <String, dynamic>{
+      'PaymentCyclePeriod': instance.paymentCyclePeriod,
+      'PaymentCycleFrq': instance.paymentCycleFrq,
+      'PaymentCycle': instance.paymentCycle,
+    };
+
+BillingBuyData _$BillingBuyDataFromJson(Map<String, dynamic> json) =>
+    BillingBuyData(
+      payResult: json['payResult'] as String? ?? '',
+      payDetails: json['payDetails'] as String? ?? '',
+    );
+
+Map<String, dynamic> _$BillingBuyDataToJson(BillingBuyData instance) =>
+    <String, dynamic>{
+      'payResult': instance.payResult,
+      'payDetails': instance.payDetails,
+    };
+
+GetUserPurchaseListAPIResult _$GetUserPurchaseListAPIResultFromJson(
+        Map<String, dynamic> json) =>
+    GetUserPurchaseListAPIResult(
+      cPResult: json['CPResult'] as String? ?? '',
+      cPStatus: json['CPStatus'] as String? ?? '',
+      invoiceDetails: (json['invoiceDetails'] as List<dynamic>?)
```

Comment:
You may need to regenerate this file.
```suggestion
      invoiceDetails: (json['InvoiceDetails'] as List<dynamic>?)
```

Reply:
fixed.